app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -53,6 +53,7 @@ class ManageIQ::Providers::Amazon::CloudManager < ManageIQ::Providers::CloudMana
 
   delegate :cloud_volumes,
            :cloud_volume_snapshots,
+           :supports_create,
```

Comment:
I don't think you'll want this. This would mean `Amazon::CloudManger.supports_create? #=> true`